### PR TITLE
Initial api dev

### PR DESCRIPTION
**Pull request by joseferus**
Add my changes to the api to the main branch

### CODE REVIEW

File: api/DBrouters/dbRouter.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
+// const { Pool } = require('pg');
```
Comment (NoahCardoza) [inline api/DBrouters/dbRouter.js:1]: Great setup in the absence of a DB! This will make it much easier once I hook up the DB.

File: api/index.js [hunk 2]
```diff
@@ -2,11 +2,14 @@ const express = require('express');
...
 
+app.use('/api/users', require('./routers/api/users'));
+app.use('/api/quizzes', require('./routers/api/quizzes'));
+
 app.get('/', (_req, res) => {
   res.send('Hello World!');
 });
 
 app.listen(port, () => {
-  // eslint-disable-next-line no-console
```
Comment (NoahCardoza) [inline api/index.js:10]: I should really disable this since it's not only a front-end app.

File: api/index.js [hunk 3]
```diff
@@ -2,11 +2,14 @@ const express = require('express');
 const app = express();
 const port = 1337;
 
+app.use('/api/users', require('./routers/api/users'));
```
Comment (NoahCardoza) [inline api/index.js]: Actually, request will be rewritten so there is no need for `/api` you can just use `/users`.

**Review by NoahCardoza — CHANGES_REQUESTED** [review]
Just update the routes that start with `/api` and I think that's it!

File: api/index.js (hunk 2 above)
Comment (joseferus) [reply api/index.js:10]: so just can the whole line?

**Comment by joseferus** [comment]
I removed the "api" directory under /routes and updated the references, and by "I" I mean WebStorm

**Comment by NoahCardoza** [comment]
Sorry if I wasn't clear, I meant in the route declarations:

image

**Comment by joseferus** [comment]
there we go, lol

**Comment by joseferus** [comment]
whoops

**Comment by NoahCardoza** [comment]
I was about to say.. wat? 😂

**Review by NoahCardoza — APPROVED** [review]

**Pull request merged**